nvflare/app_opt/tracking/mlflow/mlflow_receiver.py

```diff
@@ -0,0 +1,324 @@
+from nvflare.app_common.widgets.streaming import AnalyticsReceiver
+
+
+class MlflowConstants:
+    EXPERIMENT_TAG = "experiment_tag"
+    RUN_TAG = "run_tag"
+    EXPERIMENT_NAME = "experiment_name"
+
+
+class MLflowReceiver(AnalyticsReceiver):
+    def __init__(
+        self,
+        tracking_uri: Optional[str] = None,
+        kwargs: Optional[dict] = None,
+        artifact_location: Optional[str] = None,
+        events=None,
+        buffer_flush_time=1,
+    ):
+        """MLflowReceiver receives log events from clients and deliver them to the MLflow tracking server.
+
+        Args:
+            tracking_uri (Optional[str], optional): MLflow tracking server URI. When this is not specified, the metrics will be written to the local file system.
+                                If the tracking URI is specified, the MLflow tracking server must started before running the job. Defaults to None.
+            kwargs (Optional[dict], optional): keyword arguments:
+               "experiment_name" (str): Specifies the experiment name. If not specified, the default name of "FLARE FL Experiment" will be used.
+               "run_name" (str): Specifies the run name
+               "experiment_tags" (dict): Tags used when creating the MLflow experiment.
+                                  "mlflow.note.content" is a special MLflow tag. When provided, it displays as experiment
+                                  description field on the MLflow UI. You can use Markdown syntax for the description.
+               "run_tags" (str): Tags used when creating the MLflow run. "mlflow.note.content" is a special MLflow tag.
+                                  When provided, it displays as run description field on the MLflow UI.
+                                  You can use Markdown syntax for the description.
+            artifact_location (Optional[str], optional): Relative location of artifacts. Currently only text is supported at the moment.
+            events (_type_, optional): The event the receiver is listening to. By default, it listens to "fed.analytix_log_stats".
+            buffer_flush_time (int, optional): The time in seconds between deliveries of event data to the MLflow tracking server. The
+                                       data is buffered and then delivered to the MLflow tracking server in batches, and
+                                       the buffer_flush_time controls the frequency of the sending. By default, the buffer
+                                       flushes every second. You can reduce the time to a fraction of a second if you prefer
+                                       less delay. Keep in mind that reducing the buffer_flush_time will potentially cause high
+                                       traffic to the MLflow tracking server, which in some cases can actually cause more latency.
+        """
+        if events is None:
+            events = ["fed.analytix_log_stats"]
```

Comment:
This "analytix_log_stats" we can use the constants we defined?

I am not sure if we want to have this default to be this or not.
This default only works IF client side has a Writer and a ConvertToFedevent component with prefix "fed."

Reply:
Since we need something to be the default, I think for now, this is okay. I changed it to use a constant.